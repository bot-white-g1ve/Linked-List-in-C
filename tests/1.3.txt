NEW 5

 
1
1e-1
a
TYPE 0