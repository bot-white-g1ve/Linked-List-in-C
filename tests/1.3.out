List 0:  ->   -> 1 -> 0.10 -> a
string -> char -> int -> float -> char
